src/modules/integrator-guide/pages/compose-configuration/index.adoc

```diff
@@ -867,39 +867,17 @@ length(title) < 5
 
 == Page Block Magnification
 
-Page blocks have a magnification option that allows the user to zoom in order to inspect the details of the block. 
-This is useful because it allows the user to examine the page block in distraction-free mode and focus on that specific block in particular.
-
-=== Magnification Options
-
-.Page Block Magnification Options:
-[cols="1s,5a"]
-|===
-
-| [#tpl-syntax-fref-disabled]#<<tpl-syntax-fref-disabled,Disabled>>#
-|
-Block magnification is disabled.
-
-| [#tpl-syntax-fref-modal]#<<tpl-syntax-fref-modal,Modal>>#
-|
-The block magnification is in the form of a modal.
-
-| [#tpl-syntax-fref-fullscreen]#<<tpl-syntax-fref-fullscreen,Fullscreen>>#
-|
-The block magnification is in the form of a full-width and full-height modal.
-
-|===
-
-image:compose-configuration/enable-magnification.png[Page Block Magnification Options, role="data-zoomable"]
-
-=== Enabling Page Block Magnification
-
+Page blocks have a magnification option that allows the user to zoom in order to inspect the details of the block in distraction-free mode. 
 By default, block magnification is disabled. 
-When set, blocks that have this configuration will have a zoom-in icon in the top right corner of the block, allowing the user to magnify the block upon clicking on the icon. 
+When set, blocks that have this configuration will have a zoom-in icon in the top right corner of the block, allowing the user to magnify the block upon clicking on the icon.
+
+=== Configuring Page Block Magnification
 
 To display the magnification icon and enable block magnification, navigate to the page builder, choose a block, and go to its general tab. 
 There you will find the block magnification options as a select dropdown.
 
+image:compose-configuration/enable-magnification.png[Page Block Magnification Options, role="data-zoomable"]
```

Comment:
Add an annotation around the section.
Here is how Katrin has done it https://github.com/cortezaproject/corteza-docs/pull/383/files#diff-f242514d6f455c473b9713de6c40271db734ba9139cde1a18406926cec652e70R731